quinn-proto/src/transport_parameters.rs

```diff
@@ -381,6 +392,12 @@ impl TransportParameters {
                     }
                     params.max_datagram_frame_size = Some(r.get().unwrap());
                 }
+                0x2ab2 => {
+                    if len != 0 {
+                        return Err(Error::Malformed);
+                    }
+                    params.grease_quic_bit = true;
```

Comment:
Nit: `match`?

Reply:
I don't see a way to use `match` here that doesn't add more syntatic noise. Did you have something specific in mind?

Reply:
```rust
match len {
    0 => params.grease_quic_bit = true,
    _ => return Err(Error::Malformed),
}
```
?

Reply:
Ah, my brain has trouble putting side-effect-oriented code in value positions like that :D

Reply:
Statements are just expressions that yield () as far as Rust is concerned. 😎